mpa/stage.py

```diff
@@ -325,3 +326,29 @@ def read_label_schema(ckpt_path, name_only=True, file_name='label_schema.json'):
         else:
             all_classes = []
         return all_classes
+
+    @staticmethod
+    def get_dataset_without_wrapping(data_cfg):
+        times = 1
+        target_dataset = data_cfg
+        while hasattr(target_dataset, 'dataset'):
+            if hasattr(target_dataset, 'times'):
+                times = target_dataset.times
+            target_dataset = target_dataset.dataset
+        return target_dataset, times
```

Comment:
Why don't you merge get_train_data_cfg() & get_dataset_withou_wrapping()?
Or, at least get_train_data_cfg() could be more simplified by just calling this function.